package.json

```diff
@@ -1555,7 +1555,12 @@
         "latex-workshop.intellisense.includegraphics.preview.enabled": {
           "type": "boolean",
           "default": true,
-          "markdownDescription": "Enable preview for `\\includegraphics` completion."
+          "markdownDescription": "Enable preview for `\\includegraphics`."
```

Comment:
The variable `intellisense.includegraphics.preview.enabled` is used in 

https://github.com/James-Yu/LaTeX-Workshop/blob/63ca5b80ea09c7fb7ac8b4ce70aa6e5c700acd6f/src/providers/completion.ts#L105-L112

https://github.com/James-Yu/LaTeX-Workshop/blob/e38c440a63b61c71b18a2d43dbbc1cfdfa9740e8/src/providers/completer/input.ts#L163-L169

This deals with preview of completion items. So the description should not be changed.